### PR TITLE
Add quota behat test

### PR DESCRIPTION
**Pull request by MorrisJobke**
- adds a test for a failing upload if a user has a quota of 0
- adds a parameter to `run.sh` to run single behat tests
- fixes code styles

cc @DeepDiver1975 @SergioBertolinSG

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Failures are unrelated:

```

--- Failed scenarios:

    /ssd/jenkins/workspace/ocs-api-integration-tests-ci/build/integration/features/provisioning-v1.feature:5
```

@DeepDiver1975 Can we retrigger the tests anyway?

File: build/integration/features/bootstrap/Provisioning.php [hunk 1]
```diff
@@ -511,6 +511,20 @@ public function appIsEnabled($app) {
 	}
 
 	/**
+	 * @Given user :user has a quota of :quota
```
Comment (MorrisJobke) [inline build/integration/features/bootstrap/Provisioning.php]: Easier to read syntax of the PHPDoc that is also supported by behat ;)

Comment (MorrisJobke) [inline build/integration/features/bootstrap/Provisioning.php]: cc @DeepDiver1975 @SergioBertolinSG

**Comment by MorrisJobke** [comment]
I rebased to retrigger the CI run.

File: build/integration/run.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ echo $PHPPID_FED
 export TEST_SERVER_URL="http://localhost:$PORT/ocs/"
 export TEST_SERVER_FED_URL="http://localhost:$PORT_FED/ocs/"
 
-vendor/bin/behat -f junit -f pretty
+vendor/bin/behat -f junit -f pretty $1
```
Comment (DeepDiver1975) [inline build/integration/run.sh]: :question:

Comment (MorrisJobke) [inline build/integration/run.sh]: allows `./run.sh features/webdav-related.feature` which only run the behat tests from this file ;)

Comment (MorrisJobke) [inline build/integration/run.sh]: or:

`./run.sh features/webdav-related.feature:12´ which only runs the scenario in line 12 of that file ;)

Comment (DeepDiver1975) [inline build/integration/run.sh]: THX

**Comment by rullzer** [comment]
More tests :) awesome!

:+1:

**Comment by SergioBertolinSG** [comment]
I was editing the run.sh only locally to select the test, better to have it as argument.

:+1:

**Pull request merged**